diff --git a/node_modules/expo-localization/ios/LocalizationModule.swift b/node_modules/expo-localization/ios/LocalizationModule.swift
index 42a7476..3a18584 100644
--- a/node_modules/expo-localization/ios/LocalizationModule.swift
+++ b/node_modules/expo-localization/ios/LocalizationModule.swift
@@ -112,6 +112,8 @@ public class LocalizationModule: Module {
       return "roc"
     case .iso8601:
       return "iso8601"
+    default:
+      return "gregory"
     }
   }
 
